templates/conf/fluent.conf.erb

```diff
@@ -32,13 +32,14 @@
    index_name "#{ENV['FLUENT_ELASTICSEARCH_LOGSTASH_INDEX_NAME'] || 'logstash'}"
    type_name "#{ENV['FLUENT_ELASTICSEARCH_LOGSTASH_TYPE_NAME'] || 'fluentd'}"
 <% if is_v1 %>
+   bulk_message_request_threshold "#{ENV['FLUENT_ELASTICSEARCH_BULK_MESSAGE_REQUEST_THRESHOLD'] || '10M'}"
```

Comment:
[nits] Default value looks 20M (https://github.com/uken/fluent-plugin-elasticsearch/blob/v3.5.1/lib/fluent/plugin/out_elasticsearch.rb#L56). Was this done on purpose?

Reply:
Yes, done on purpose, though now that I know more about it I would default it to 4M. 20M seems to have been set somewhat arbitrarily, whereas I am concerned about the 10M request size limit on AWS Elasticsearch. See https://github.com/uken/fluent-plugin-elasticsearch/issues/588 for details.